src/core_write.cpp

```diff
@@ -205,6 +210,13 @@ void TxToUniv(const CTransaction& tx, const uint256& hashBlock, UniValue& entry,
                 }
                 in.pushKV("txinwitness", txinwitness);
             }
+            if (txundo) {
+                CAmount nPrevOut = txundo->vprevout[i].out.nValue;
+                // Sanity check to avoid overflow
+                if (MoneyRange(nPrevOut)) {
+                    nTotalIn += nPrevOut;
+                }
```

Comment:
This seems unnecessary.

Reply:
Yeah, I agree it seems unnecessary.  I added this in order to fix a failing fuzz test.